{
  "pci_instances_actions_delete_instance_title": "Elimina un’istanza",
  "pci_instances_actions_delete_instance_confirmation_message": "Vuoi davvero eliminare l'istanza {{ name }}?",
  "pci_instances_actions_delete_instance_error_message": "Si è verificato un errore durante l'eliminazione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_delete_instance_success_message": "Eliminazione dell’istanza {{ name }} in corso. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_stop_instance_title": "Spegnere l’istanza",
  "pci_instances_actions_stop_instance_confirmation_message": "Stai per spegnere l’istanza {{ name }}. Le risorse dedicate all'istanza Public Cloud sono sempre riservate (indirizzo IP incluso). Puoi riavviare l'istanza in qualsiasi momento. Nel frattempo, l’importo fatturato resta invariato.",
  "pci_instances_actions_stop_instance_error_message": "Si è verificato un errore durante lo spegnimento dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_stop_instance_success_message": "L’istanza {{ name }} è stata arrestata. Le risorse dedicate sono sempre riservate (indirizzo IP incluso) e l’importo fatturato rimane invariato. <0>Maggiori informazioni</0>",
  "pci_instances_actions_soft_reboot_instance_title": "Riavvia a caldo (soft)",
  "pci_instances_actions_soft_reboot_instance_confirmation_message": "Stai per riavviare l'istanza {{ name }}.",
  "pci_instances_actions_soft_reboot_instance_error_message": "Si è verificato un errore durante il riavvio dell’istanza {{ instance }}: {{ error }}",
  "pci_instances_actions_soft_reboot_instance_success_message": "Il riavvio dell’istanza {{ name }} è iniziato. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_hard_reboot_instance_error_message": "Si è verificato un errore durante il riavvio dell’istanza {{ instance }}: {{ error }}",
  "pci_instances_actions_hard_reboot_instance_success_message": "Il riavvio dell’istanza {{ name }} è iniziato. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_hard_reboot_instance_title": "Riavvia a freddo (hard)",
  "pci_instances_actions_hard_reboot_instance_confirmation_message": "Stai per riavviare l'istanza {{ name }}.",
  "pci_instances_actions_rescue_start_instance_title": "Riavvia in modalità Rescue",
  "pci_instances_actions_rescue_start_instance_confirmation_message": "Selezionando l’opzione “Distribuzione Rescue Made-in-OVH”, riceverai una password temporanea per accedere alla tua istanza in modalità Rescue. In caso contrario, l’istanza sarà accessibile con la tua chiave SSH abituale ma l'impronta SSH dell'istanza sarà diversa. Si è verificato un errore durante la reinstallazione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_rescue_start_instance_error_message": "Si è verificato un errore durante la reinstallazione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_rescue_start_instance_info_message": "Per accedere al server {{name}} in modalità Rescue, utilizza <0>ssh root@{{ip}}</0> con la password visibile nella console VNC o tramite <0>openstack console log show {{name}}</0>. Attenzione: solo il disco principale è accessibile con questa modalità.",
  "pci_instances_actions_rescue_start_instance_select_image_label": "La tua immagine:",
  "pci_instances_actions_rescue_start_instance_success_message": "Riavvio dell’istanza {{ name }} in corso. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_rescue_start_instance_rescue_image_label": "Distribuzione Rescue Made-in-OVH",
  "pci_instances_actions_rescue_end_instance_title": "Esci dalla modalità Rescue",
  "pci_instances_actions_rescue_end_instance_confirmation_message": "Stai per riavviare l'istanza {{ name }}.",
  "pci_instances_actions_rescue_end_instance_error_message": "Si è verificato un errore durante la reinstallazione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_rescue_end_instance_success_message": "Riavvio dell’istanza {{ name }} in corso. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_reinstall_instance_title": "Reinstalla un’istanza",
  "pci_instances_actions_reinstall_instance_confirmation_message": "Vuoi davvero reinstallare l’istanza {{ name }}?",
  "pci_instances_actions_new_image_reinstall_instance_confirmation_message": "L'immagine di sistema attualmente utilizzata è obsoleta e non è più supportata. Reinstalla l’ambiente con una nuova immagine tra quelle disponibili nel seguente elenco",
  "pci_instances_actions_reinstall_instance_warning_message": "Tutti i dati verranno eliminati.",
  "pci_instances_actions_reinstall_instance_error_message": "Si è verificato un errore durante la reinstallazione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_reinstall_instance_success_message": "Reinstallazione dell’istanza {{ name }} in corso. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_backup_instance_title": "Crea un backup",
  "pci_instances_actions_backup_instance_name_label": "Inserisci il nome del tuo backup:",
  "pci_instances_actions_backup_instance_price": "Prezzo: ~{{price}}/mese/GB",
  "pci_instances_actions_backup_instance_warning_message": "La tua istanza sarà sospesa per qualche secondo. Le operazioni di lettura/scrittura del disco potrebbero essere rallentate fino al completamento del backup",
  "pci_instances_actions_backup_instance_error_message": "Si è verificato un errore durante il backup dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_backup_instance_success_message": "Creazione del backup {{ name }} in corso.",
  "pci_instances_actions_billing_monthly_activate_instance_title": "Passa al forfait mensile",
  "pci_instances_actions_billing_monthly_activate_instance_confirmation_message": "<0>Riduci l'importo della tua fattura attivando il forfait mensile sulla tua istanza.</0> <1>Visualizza i prezzi</1>",
  "pci_instances_actions_billing_monthly_activate_instance_warning_message": "La tua istanza sarà sospesa per qualche secondo. Le operazioni di lettura/scrittura del disco potrebbero essere rallentate fino al completamento del backup",
  "pci_instances_actions_billing_monthly_activate_instance_error_message": "Si è verificato un errore durante l’attivazione del forfait mensile sull’istanza {{ name }}: {{ error }}",
  "pci_instances_actions_billing_monthly_activate_instance_success_message": "Sull’istanza {{ name }} è ora attivo il forfait mensile.",
  "pci_instances_actions_start_instance_title": "Avvio dell’istanza",
  "pci_instances_actions_start_instance_confirmation_message": "Stai per avviare l'istanza {{ name }}.",
  "pci_instances_actions_start_instance_error_message": "Si è verificato un errore durante l’avvio dell’istanza {{ name }}: {{ error }}",
  "pci_instances_actions_start_instance_success_message": "Avvio dell’istanza {{ name }} in corso. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_instance_unknown_error_message": "Si è verificato un errore e non è possibile accedere alla tua richiesta: {{ error }}",
  "pci_instances_actions_shelve_instance_title": "Sospensione dell'istanza",
  "pci_instances_actions_shelve_instance_success_message": "Sospensione dell’istanza {{ name }} in corso. Le risorse dedicate sono rese nuovamente disponibili, ma l'indirizzo IP viene conservato. I dati presenti sul disco locale sono stati salvati in uno snapshot, il cui costo verrà addebitato in fattura. <0>Maggiori informazioni</0>",
  "pci_instances_actions_unshelve_instance_title": "Riattiva la tua istanza",
  "pci_instances_actions_unshelve_instance_success_message": "Riattivazione dell’istanza {{ name }} in corso. Ti chiediamo di attendere, l'operazione potrebbe richiedere alcuni minuti.",
  "pci_instances_actions_unshelve_instance_error_message": "Si è verificato un errore durante la riattivazione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_unshelve_instance_confirmation_message": "Stai per riattivare l'istanza {{ name }}. Le risorse dedicate all'istanza Public Cloud saranno ripristinate (indirizzo IP incluso). I dati del disco locale saranno ripristinati, la durata dell'operazione dipende dalla dimensione del disco locale.",
  "pci_instances_actions_shelve_instance_error_message": "Si è verificato un errore durante la sospensione dell'istanza {{ name }}: {{ error }}",
  "pci_instances_actions_shelve_instance_confirmation_message": "Stai per sospendere l'istanza {{ name }}. Le risorse dedicate all'istanza Public Cloud verranno liberate ma l’IP verrà mantenuto. I dati del disco locale saranno salvati e la fatturazione verrà applicata solo allo storage utilizzato per questo fine. La durata dell'operazione varia in base alla dimensione del disco locale. È possibile riattivare l'istanza in qualsiasi momento.",
  "pci_instances_actions_shelve_instance_nota_message": "(1) Attenzione: per le istanze con fatturazione mensile, la fatturazione classica continuerà a essere applicata indipendentemente dallo stato del servizio.",
  "pci_instances_actions_instance_unavailable_action_error_message": "Questa azione non è disponibile per l’istanza {{ name }}",
  "pci_instances_actions_instance_success_message": "Lo stato della tua istanza {{ name }} è stato aggiornato correttamente.",
  "pci_instances_actions_instance_network_change_dns": "Modifica il reverse DNS ",
  "pci_instances_actions_instance_network_activate_mitigation": "Attiva/Disattiva la mitigazione",
  "pci_instances_actions_instance_network_firewall_settings": "Configura il firewall",
  "pci_instances_actions_instance_network_network_settings": "Gestisci le reti private",
  "pci_instances_actions_instance_network_network_attach": "Associa una rete",
  "pci_instances_actions_instance_network_network_attach_title": "Associa una o più reti private alla tua istanza",
  "pci_instances_actions_instance_network_network_empty_message": "Al momento non sono presenti reti private compatibili a cui associare questa istanza.",
  "pci_instances_actions_instance_network_network_attach_success_message": "La rete privata {{ network }} è stata associata all’istanza {{ instance }}.",
  "pci_instances_actions_instance_attach_error_message": "Si è verificato un errore durante l’aggiornamento dell’istanza: {{ message }} ",
  "pci_instances_actions_instance_volume_attach_title": "Associa un volume alla tua istanza",
  "pci_instances_actions_instance_volume_attach_empty_message": "Nessun volume disponibile nella Region della tua istanza.",
  "pci_instances_actions_instance_volume_attach_selector_label": "Associare una risorsa",
  "pci_instances_actions_instance_volume_attach_select": "Selezionare un volume esistente",
  "pci_instances_actions_instance_volume_attach_information": "Puoi associare un volume esistente o creare un nuovo volume da associare alla tua istanza. ",
  "pci_instances_actions_instance_volume_attach_create_link": "Creare un volume",
  "pci_instances_actions_instance_volume_attach_success_message": "Il volume {{ volume }} è stato associato all’istanza {{ instance }}."
}
